bitcoind-rpc/bitcoind-rpc.sbt

```diff
@@ -51,7 +51,10 @@ TaskKeys.downloadBitcoind := {
     val location =
       if (version == experimentalVersion)
         s"https://s3-us-west-1.amazonaws.com/suredbits.com/bitcoin-core-$version/bitcoin-$version-$platform.$suffix"
-      else
+      else if (version.init.endsWith("rc")) { // if it is a release candidate
```

Comment:
i think this can be removed now?

Reply:
imo we should leave this, might want to use a release canidate in the future and will just have to put this back